weave-js/src/components/Form/TextField.tsx

```diff
@@ -78,34 +79,37 @@ export const TextField = ({
     <Tailwind style={{width: '100%'}}>
       <div
         className={classNames(
-          'relative',
-          textFieldSize === 'medium' ? 'h-32' : 'h-40'
+          'relative rounded-sm',
+          textFieldSize === 'medium' ? 'h-32' : 'h-40',
+          'outline outline-1 outline-moon-250',
+          {
+            'hover:outline-2 [&:hover:not(:focus-within)]:outline-[#83E4EB]':
+              !errorState,
+            'focus-within:outline-2 focus-within:outline-teal-400': !errorState,
```

Comment:
the reason outline wasn't working when input was focused was because we [were using](https://github.com/wandb/weave/pull/2056/files#diff-cfe478a74185bdf97c39dd00bfa08d181cbf7bb6e9d24630ebad18b8f1b17bacL93) `:focus`, but `divs` don't have a focus state. the element being focused is the inner `input`. we must use `:focus-within` instead